edu-services/gradebook-service/hibernate/src/hibernate/org/sakaiproject/tool/gradebook/GradableObject.hbm.xml

```diff
@@ -45,6 +45,7 @@
 			<property name="countNullsAsZeros" column="IS_NULL_ZERO" type="java.lang.Boolean"/>
 			<property name="hideInAllGradesTable" column="HIDE_IN_ALL_GRADES_TABLE" type="java.lang.Boolean"/>
 			<property name="externalData" column="EXTERNAL_DATA" type="text" />
+                        <property name="createTask" column="CREATE_TASK" type="java.lang.Boolean" />
```

Comment:
I don't think this is a good approach, coupling services together at a database level like this. You should just be able to call the task service to work out whether to check a box in the UI or not.